+srv/expServer.m

```diff
@@ -357,50 +344,81 @@ function calibrateWaterDelivery()
   end
 
   function whiteScreen()
+    % WHITESCREEN Change screen background to white
```

Comment:
Can we consistently write docstrings for functions as what the function does? So in this case,
`% WHITESCREEN changes screen background to white.`
so that it reads as a sentence? For classes, can we use a metaphor in the first line of the docstring? I do the same in the example function and class docstrings I showed you.

Reply:
That doesn't appear to be the way MATLAB does it (e.g. 'str2double Convert string to double precision value') but you can pick whatever format you like